v4/newrelic/log.go

```diff
@@ -26,18 +25,12 @@ type Logger interface {
 	DebugEnabled() bool
 }
 
-// NewLogger creates a basic Logger at info level.
-//
-// Deprecated: NewLogger is deprecated and will be removed in a future release.
-// Use the ConfigInfoLogger ConfigOption instead.
-func NewLogger(w io.Writer) Logger {
+// newLogger creates a basic Logger at info level.
+func newLogger(w io.Writer) Logger {
```

Comment:
I changed this to non-exported because it was already deprecated in the v3 agent.